src/rabbit_channel.erl

```diff
@@ -1979,3 +1987,7 @@ erase_queue_stats(QName) ->
     [erase({queue_exchange_stats, QX}) ||
         {{queue_exchange_stats, QX = {QName0, _}}, _} <- get(),
         QName0 =:= QName].
+
+get_vhost(State = #ch{virtual_host = VHost}) -> VHost.
```

Comment:
in both these getters `State` is not used and it gives a Warning while compiling